test/unit/parser_test.rb

```diff
@@ -685,14 +685,17 @@ def test_round_with_literal
   end
 
   def test_round_with_field
-    filter = "ListPrice Eq round(FieldName)"
```

Comment:
A few of these searches were invalid. The syntax correctly broke those tests.

Reply:
So, what was wrong here was having a field on the right hand side of the expression?

Reply:
From what I could tell, we don't currently have logic for handling fields on the right hand side of an expression. All the tests I ran ended up with a 500 error.